Add Hot Pot restaurant preset

novolife opened the pull request:
### Description, Motivation & Context

Hot pots are widely popular in East and Southeast Asia, with types such as Beijing hot pot, Sichuan hot pot, Teochew hot pot, shabu-shabu, sukiyaki and etc. This is a very common POI, but the data is currently low because there is no preset, so not easy to find, adding this preset will help the POI data improve.

This PR also sorts strings of `cuisine` according to the classification on the wiki and A-Z.

### Related issues

#1521 

### Links and data

**Relevant OSM Wiki links:**

https://wiki.openstreetmap.org/wiki/Key:cuisine

https://wiki.openstreetmap.org/wiki/Tag:cuisine%3Dhotpot

**Relevant tag usage stats:**

 730

### Checklist and Test-Documentation Template

<details><summary>Read on to get your PR merged faster…</summary>

Follow these steps to test your PR yourself and make it a lot easier and faster for maintainers to check and approve it.

**This is how it works:**
1. After you submit your PR, the system will create a preview and comment on your PR:
   > 🍱 You can preview the tagging presets of this pull request here.
   If this is your first contribution to this project, the preview will not happen right away but requires a click from one of the project members. We will do this ASAP.

2. Once the preview is ready, use it to test your changes.

3. Now copy the snippet below into a new comment and fill out the blanks.

4. Now your PR is ready to be reviewed.

```
## Test-Documentation

### Preview links & Sidebar Screenshots

<!-- Use the preview to find examples, select the feature in question and **copy this link here**.
     Find examples of nodes/areas. Find examples with a lot of tags or very few tags. – Whatever helps to test this thoroughly.
     Add relevant **screenshots** of the sidebar of those examples. -->

<!-- FYI: What we will check:
     - Is the [icon](https://github.com/ideditor/schema-builder/blob/main/ICONS.md) well chosen.
     - Are the fields well-structured and have good labels.
     - Do the dropdowns (etc.) work well and show helpful data. -->

### Search

<!-- **Test the search** of your preset and share relevant **screenshots** here.
     - Test the preset name as search terms.
     - Also test the preset terms and aliases as search terms (if present). -->

### Info-`i`

<!-- **Test the info-i** for your fields and preset and share relevant **screenshots** here.
     The info needs to help mappers understand the preset and when to use it.
     [Learn more…](https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md#info-i)
 -->

### Wording

- [ ] American English
- [ ] `name`, `aliases` (if present) use Title Case
- [ ] `terms` (if present) use lower case, sorted A-Z
<!-- Learn more in https://github.com/openstreetmap/id-tagging-schema/blob/main/GUIDELINES.md#2-design-the-preset -->
```

</details>

novolife committed:
add strings for cuisine field

novolife committed:
Add Hot Pot restaurant preset

novolife committed:
revert commit 798c796a7c451cf963f0902c875115479f58c3f1.

novolife commented:
![ss4](https://github.com/user-attachments/assets/fdd96f01-485f-4f4c-94c4-63ac09d54da1)

tordans reviewed (changes requested):
Technically this looks good.
I am not sure about the long list of terms, but LLMs say they are OK…

But what I am concered with is that this preset will only target a few results.
Look at https://overpass-turbo.eu/s/25zi (general usage) vs. https://overpass-turbo.eu/s/25zj (what this PR targets).

I suggest to only add the option to the cuisine field but not a separate preset.

But lets see what others think.


Test links
* https://pr-1569--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/52.35702/4.89054&disable_features=boundaries&background=Bing&id=n2165936476
* with NSI https://pr-1569--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/50.10750/8.66564&disable_features=boundaries&background=Bing&id=n2144172795
* general restaurant https://pr-1569--ideditor-presets-preview.netlify.app/id/dist/#locale=en&map=20.00/48.13588/11.55603&disable_features=boundaries&background=Bing&id=n1713202488

novolife committed:
change name

matkoniecz commented:
@tordans Have you looked at tagging in Asia, say Taiwan?

tordans commented on data/presets/amenity/restaurant/hotpot.json:
```diff
@@ -0,0 +1,40 @@
+        "teoswa",
+        "yao hon"
+    ],
+    "tags": {
+        "amenity": "restaurant",
+        "cuisine": "hotpot"
+    },
+    "reference": {
+        "key": "cuisine",
+        "value": "hotpot"
+    },
```
```suggestion
    "tags": {
        "cuisine": "hotpot"
    },
    "addTags": {
        "amenity": "restaurant",
        "cuisine": "hotpot"
    },
```

As far as I see the only usage of `cuisine=hotpot` is in combination with `amenity=restaurant`. See below.
We can therefore change this around like so…

<details><summary>Data on amenity tags for this cuisine; data on other tags for this cuisine</summary>
<p>


```
% curl -g https://postpass.geofabrik.de/api/0.2/interpreter \
  --data-urlencode "options[geojson]=false" \
  --data-urlencode "data=
  SELECT
    value AS amenity,
    COUNT(*) AS count
  FROM (
    SELECT
      (jsonb_each_text(tags)).key AS key,
      (jsonb_each_text(tags)).value AS value
    FROM postpass_pointpolygon
    WHERE tags->>'cuisine' = 'hotpot'
  ) AS kv
  WHERE key = 'amenity'
  GROUP BY value
  HAVING COUNT(*) > 10
  ORDER BY count DESC"



{
    "result": [
        {
            "count": 731,
            "amenity": "restaurant"
        }
    ],
    "metadata": {
        "generator": "Postpass API 0.2",
        "timestamp": "2025-09-14T15:31:51Z"
    }
}%
```

```

% curl -g https://postpass.geofabrik.de/api/0.2/interpreter \
  --data-urlencode "options[geojson]=false" \
  --data-urlencode "data=
  SELECT
    kv.key,
    COUNT(*) AS tag_count
  FROM (
    SELECT
      (jsonb_each_text(tags)).*
    FROM postpass_pointpolygon
    WHERE tags->>'cuisine' = 'hotpot'
  ) AS kv
  WHERE kv.key <> 'cuisine'
  GROUP BY kv.key
  ORDER BY tag_count DESC"

{
    "result": [
        {
            "key": "amenity",
            "tag_count": 732
        },
        {
            "key": "name",
            "tag_count": 688
        },
        {
            "key": "addr:street",
            "tag_count": 315
        },
        {
            "key": "addr:housenumber",
            "tag_count": 307
        },
        {
            "key": "addr:city",
            "tag_count": 299
        },
        {
            "key": "addr:district",
            "tag_count": 259
        },
        {
            "key": "addr:country",
            "tag_count": 241
        },
        {
            "key": "addr:postcode",
            "tag_count": 181
        },
        {
            "key": "phone",
            "tag_count": 160
        },
        {
            "key": "name:en",
            "tag_count": 148
        },
        {
            "key": "opening_hours",
            "tag_count": 144
        },
        {
            "key": "name:zh",
            "tag_count": 133
        },
        {
            "key": "branch",
            "tag_count": 103
        },
        {
            "key": "brand",
            "tag_count": 100
        },
        {
            "key": "wheelchair",
            "tag_count": 68
        },
        {
            "key": "brand:wikidata",
            "tag_count": 66
        },
        {
            "key": "brand:zh",
            "tag_count": 62
        },
        {
            "key": "brand:en",
            "tag_count": 61
        },
        {
            "key": "addr:full",
            "tag_count": 59
        },
        {
            "key": "website",
            "tag_count": 59
        },
        {
            "key": "outdoor_seating",
            "tag_count": 59
        },
        {
            "key": "addr:floor",
            "tag_count": 45
        },
        {
            "key": "operator",
            "tag_count": 41
        },
        {
            "key": "takeaway",
            "tag_count": 41
        },
        {
            "key": "air_conditioning",
            "tag_count": 40
        },
        {
            "key": "level",
            "tag_count": 37
        },
        {
            "key": "operator:en",
            "tag_count": 35
        },
        {
            "key": "addr:province",
            "tag_count": 35
        },
        {
            "key": "operator:zh",
            "tag_count": 33
        },
        {
            "key": "disused:name",
            "tag_count": 32
        },
        {
            "key": "smoking",
            "tag_count": 31
        },
        {
            "key": "operator:wikidata",
            "tag_count": 28
        },
        {
            "key": "building",
            "tag_count": 27
        },
        {
            "key": "payment:cash",
            "tag_count": 27
        },
        {
            "key": "toilets:wheelchair",
            "tag_count": 26
        },
        {
            "key": "diet:vegetarian",
            "tag_count": 22
        },
        {
            "key": "check_date",
            "tag_count": 21
        },
        {
            "key": "addr:hamlet",
            "tag_count": 19
        },
        {
            "key": "contact:facebook",
            "tag_count": 19
        },
        {
            "key": "addr:neighbourhood",
            "tag_count": 18
        },
        {
            "key": "disused:amenity",
            "tag_count": 17
        },
        {
            "key": "delivery",
            "tag_count": 17
        },
        {
            "key": "name:zh-Hant",
            "tag_count": 17
        },
        {
            "key": "indoor_seating",
            "tag_count": 16
        },
        {
            "key": "addr:state",
            "tag_count": 15
        },
        {
            "key": "addr:subdistrict",
            "tag_count": 14
        },
        {
            "key": "source",
            "tag_count": 12
        },
        {
            "key": "name:zh-Hans",
            "tag_count": 11
        },
        {
            "key": "disused:name:en",
            "tag_count": 10
        },
        {
            "key": "payment:credit_cards",
            "tag_count": 10
        },
        {
            "key": "reservation",
            "tag_count": 10
        },
        {
            "key": "wheelchair:description",
            "tag_count": 10
        },
        {
            "key": "diet:vegan",
            "tag_count": 10
        },


```

</p>
</details>

tordans reviewed (dismissed):
We can improve this a bit.

novolife committed:
Update data/presets/amenity/restaurant/hotpot.json

Co-authored-by: Tobias <t@tobiasjordans.de>

tyrasd committed:
revert to use amenity=restaurant as primary tag

tyrasd reviewed (approved):
Approving, although usage is still low, I do know from personal experience that these type of specialized restaurants are actually quite common in some parts of Asia.

tyrasd committed:
change icon

the fork-and-knife icon does not fit the preset well. bowl-and-chopsticks is also now perfect, but at least slightly better
